extensions/wikia/EditPageLayout/js/plugins/FlowTracking.js

```diff
@@ -17,25 +17,8 @@
 	WE.plugins.flowtracking = $.createClass(WE.plugin,{
 
 		initEditor: function(editor) {
-			require(['wikia.flowTracking', 'wikia.querystring', 'mw'], function(flowTrack, QueryString, mw) {
-				var namespaceId = mw.config.get('wgNamespaceNumber'),
-					articleId = mw.config.get('wgArticleId');
-
-				// Track only creating articles (wgArticleId=0) from namespace 0 (Main)
-				// IMPORTANT: on Special:CreatePage even after providing article title the namespace is set to -1 (Special Page)
-				if (namespaceId === 0 && articleId === 0) {
-					var qs = new QueryString(window.location.href);
-
-					// 'flow' is the parameter passed in the url if user has started a flow already
-					var flowParam = qs.getVal('flow', false);
-
-					if (flowParam || document.referrer) {
-						//TODO: track middle step for other flows
-					} else {
-						flowTrack.beginFlow(flowTrack.flows.CREATE_PAGE_DIRECT_URL, {editor: editorName(editor.mode)});
-					}
-				}
-
+			require(['wikia.flowTracking.createPage'], function(flowTrackCreatePage) {
```

Comment:
Out of curiosity (but also for consistency), why we shorten flowTracking (as module name) to flowTrack (as variable) in the scope of this PR?